mod_app/models/support_models.py

```diff
@@ -11,9 +11,12 @@ def __str__(self):
 
 class Link(models.Model):
```

Comment:
As discussed, I think the extra effort users might have to make to occasionally create a link twice is outweighed by the simplicity of adding Film as a foreign key on these models.

Reply:
not entirely sure I follow - the only direct link to Link is through additional_links, all others have a model that's based on Link or FileLink and they have m2m relations with the exception of video, which does have a FK relationship

Reply:
So rather than having:
```
class Film(models.Model):
    scripts = models.ManyToManyField(
            Script,
            help_text="Link to or upload script file(s)",
            related_name="scripts",
            blank=True,
        )
```
You'd have:
```
class Link(models.Model):
    url = models.URLField(blank=True, help_text="url to the item you'd like to link")
    description = models.CharField(
        max_length=250,
        help_text="short description of what the link is to (optional)",
        blank=True,
        null=True,
    )
    film = models.ForeignKey("Film",on_delete=models.CASCADE)
```

I means links can't be attached to multiple films, but makes it much easier to handle in the admin interface.